Sources/Sisyphos/Page Elements/OtherElement.swift

```diff
@@ -0,0 +1,28 @@
+public struct OtherElement: PageElement {
+    public let elementIdentifier: PageElementIdentifier
+
+    let identifier: String?
+    let label: String?
+
+    public var queryIdentifier: QueryIdentifier {
+        .init(
+            elementType: .other,
+            identifier: identifier,
+            label: label,
+            value: nil,
+            descendants: []
+        )
+    }
+
+    public init(
+        identifier: String? = nil,
+        label: String? = nil,
+        file: String = #file,
+        line: UInt = #line,
+        column: UInt = #column
+    ) {
```

Comment:
This initializer makes it possible to create an `OtherElement` that has neither a label nor an identifier.

I'm not fully decided if this is a case that we should support.

On the pro side, it might be a legimate use case that a user might have. You might need to use an element of element type `other` to exactly match an ambiguous element.

On the con side, we don't support this kind of elements in our code generation to avoid too much noise in the generated element tree. 

What do you think? Should we enforce that you can only create instances that either have a label or an identifier?

Reply:
That's a good point! In my opinion, it should support initializing with either one of them for now as it seems that the con side is more important than the pro side.

Reply:
17368d521c55810587a935efa62a8ad72e8c78d3